database_setup/aggregated-user-tracking.sql

```diff
@@ -0,0 +1,14 @@
+CREATE TABLE public.user_tracking
+(
+    username text NOT NULL,
```

Comment:
Probably a worth adding an index for username & email - we're likely to want to search using both of these at some point